### DISCUSSION TITLE
Configurazione automatica di HTTPS (Let's Encrypt) / unexpected error

### DISCUSSION THREAD

**Discussion by alecrespi78** [globaleaks/globaleaks-whistleblowing-software#3896] [Q&A] [answered]
Versione SW: 4.13.22
cercando di procedere con la configurazione automatica ricevo l'errore "errore interno del server (Unexpected)"

Ho provato a seguire le procedure indicate in questa discussione (https://github.com/orgs/globaleaks/discussions/3608) ma senza risultati.

![globaleaks-errore](https://github.com/globaleaks/GlobaLeaks/assets/89710784/8c796dd5-6129-4d8e-800b-50bc67fceb7c)
![globaleaks-https](https://github.com/globaleaks/GlobaLeaks/assets/89710784/50bd5fa6-368a-419c-96c2-a06ea3cac8f9)

**Comment by evilaliv3** [comment, marked as answer]
Vedo che avete configurato un ip al posto del dominio.

Per abilitare letsencrypt è necessario puntare un dominio verso la macchina e assicurarsi di avere aperte sugli eventuali firewall le porte 80 e 443.

**Reply by alecrespi78** [reply]
Sì l'IP che vedi corrisponde alla LAN interna.
Ho aggiunto un record dns sul dominio esterno "whistle.DOMINIO.it" che punta all'indirizzo IP pubblico esterno e alla porta relativa "IPpubblico:8443 >>> IPlocale:80".

Internamente "IPlocale" si collega senza problemi.

Magari questi 2 problemi sono collegati:
(https://github.com/orgs/globaleaks/discussions/3895)

**Comment by alecrespi78** [comment]
Posso modificare il campo "hostname" senza problemi?
Tramite indirizzo IP interno cambia qualcosa o continuerà a funzionare senza problemi?

**Reply by evilaliv3** [reply]
Una volta che verrà configurato un hostname il sistema risponderà solo sull'hostname.
